### PR TITLE
OSSL_STORE: Add OSSL_STORE_vctrl()

### PR DESCRIPTION
**Pull request by levitte**
It's a convenient complement to OSSL_STORE_ctrl()

Suggested by Norm Green

### CODE REVIEW

**Comment by mspncp** [comment]
AppVeyor complains with justification, that `OSSL_STORE_vctrl` is missing in libcrypto.num. The rest looks good to me. It's always good for reusability to have an explicit `va_list` equivalent for a three-dot function.

**Comment by levitte** [comment]
Fixup added

File: crypto/store/store_lib.c [hunk 1]
```diff
@@ -109,16 +109,22 @@ OSSL_STORE_CTX *OSSL_STORE_open(const char *uri, const UI_METHOD *ui_method,
 int OSSL_STORE_ctrl(OSSL_STORE_CTX *ctx, int cmd, ...)
 {
     va_list args;
-    int ret = 0;
+    int ret;
 
     va_start(args, cmd);
-    if (ctx->loader->ctrl != NULL)
-        ret = ctx->loader->ctrl(ctx->loader_ctx, cmd, args);
+    ret = OSSL_STORE_vctrl(ctx->loader_ctx, cmd, args);
```
[the remark this answers is not in the document]
Comment (mspncp) [reply crypto/store/store_lib.c]: I still see a `ctx->loader_ctx` here. Shouldn't this be just `ctx`?

[the remark this answers is not in the document]
Comment (levitte) [reply crypto/store/store_lib.c]: I *did* fix it! ... but in the wrong spot, so not so much.  Another fixup coming up

**Review by mspncp — APPROVED** [review]
Now it looks right to me. Approved, provided the CIs pass.

**Comment by levitte** [comment]
Hmmmm, CIs are sluggish this evening...

**Comment by levitte** [comment]
Finally

**Comment by levitte** [comment]
Merged. 4fd39122e51c3c44fcb72fb9b732d5008efda399

**Pull request closed without merging**